wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/README.md

```diff
@@ -0,0 +1,71 @@
+# WordPress.org Openverse Theme
+
+This theme loads Openverse in an `iframe` under the WordPress.org site.
+
+## Development
+
+Follow these steps to set up a local playground for the theme:
+
+0.  Install all the prerequisites.
+
+    1.  **Required:** Node.js.
+    2.  **Recommended:** Docker (to use the automatic setup)
+
+1.  Build the parent theme WordPress.org theme.
+
+    1.  `cd` into the parent theme directory at `../wporg`.
+    2.  Install all the required `npm` packages.
+        ```bash
+        $ npm install
+        ```
+    3.  Build the theme assets.
+        ```bash
+        $ npm run build
+        ```
+
+2.  You can choose to set up a new environment automatically or work in an
+    existing environment with manual setup.
+
+    **Automatic:**  
+    You can set up the development environment automatically using
+    `@wordpress/env`. _This requires Docker._
+
+    1.  Install `@wordpress/env` globally as an `npm` package.
+        ```bash
+        $ npm install -g @wordpres/env@latest
```

Comment:
typo: wordpres -> wordpress

Reply:
Nice catch! In retrospect, should have copied from the terminal instead of typing 😆.

Reply:
Resolved in 772a211.